src/common/config.ts

```diff
@@ -35,16 +35,16 @@ export const OLD_VOTING_CONTRACT_ADDRESSES = [
 ];
 
 export default function config(network: Network | null) {
-  // const infuraRpc = `https://${
-  //   network ? network?.name : "mainnet"
-  // }.infura.io/v3/${infuraId}`;
+  const infuraRpc = `https://${
+    network ? network?.name : "mainnet"
+  }.infura.io/v3/${infuraId}`;
 
   return {
     onboardConfig: {
       apiKey: process.env.REACT_APP_PUBLIC_ONBOARD_API_KEY || "",
       onboardWalletSelect: {
         wallets: [
-          { walletName: "metamask", preferred: true },
+          { walletName: "metamask" },
```

Comment:
I have removed `preferred: true` because this option hides the other wallet options behind a "show more" menu instead of making them readily visible.